types/knockout/index.d.ts

```diff
@@ -9,7 +9,6 @@
 //                 Leonardo Lombardi <https://github.com/ltlombardi>
 //                 Retsam <https://github.com/Retsam>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
```

Comment:
was removing TS version required?

Reply:
This is a feeble attempt to make it build I still have the problem of https://github.com/DefinitelyTyped/DefinitelyTyped/pull/32210#issuecomment-454794822 that I don't know how to solve :-(